docs/versions/latest/Release.md

```diff
@@ -23,12 +23,25 @@
 [//]: # (Describe the feature)
 [//]: # (Optional But higlhy recommended Specify *NONE* if missing)
 [//]: # (#### Relevant Documentation:)
+### *Enforce TLS 1.2 when Downloading PowerShell Modules*
+Added support for downloading modules in powershell polyglot project handlers through 'Ssl3' | 'Tls' | 'Tls11' | 'Tls12' | 'Tls13'
 
-
+#### Previous Behavior
```

Comment:
Notes like:

- IT didn't work
- Now it works

Don't really provide good information.

Try to explain how it was fixed and why it was failing before